src/qibo/engines/__init__.py

```diff
@@ -0,0 +1,111 @@
+from qibo import config
+from qibo.config import log, raise_error
+from qibo.engines.numpy import NumpyEngine
+from qibo.engines.tensorflow import TensorflowEngine
+
+
+def construct_backend(backend, platform=None, show_error=False):
+    def _initialize_backend(backend_cls, show_error=False):
+        try:
+            return backend_cls()
+        except (ModuleNotFoundError, ImportError):
+            if show_error:
+                name = backend_cls.__name__
+                raise_error(ModuleNotFoundError, f"{name} is not installed.")
+            return None
+
+    if backend == "qibojit":
+        from qibojit.engines import CupyEngine, NumbaEngine
```

Comment:
I believe this line will make the code crash if qibojit is not installed, right?
Maybe this could be moved inside `_initialize_backend` or wrapped around a try catch.